handbook/engineering/security/goals.md

```diff
@@ -2,38 +2,51 @@
 
-**Milestones:**
+**Milestones:** 
 
 1. We need a better way to store and access [our own secrets](https://docs.google.com/document/d/1HzO7szEm-h4fqlQOnVbcJdpDmfQiM7Rb-Tz4CMEYl-Q).
-1. We connect our [test security repository](https://github.com/sourcegraph/security-test/blob/main/README.md) to Sourcegraph Cloud and only [members who can access that repository on GitHub](https://github.com/sourcegraph/security-test/settings/access) can access that repository on Sourcegraph Cloud (i.e., [Sourcegraph organization owners](https://github.com/orgs/sourcegraph/people?query=role%3Aowner) and [@sourcegraph/security](https://github.com/orgs/sourcegraph/teams/security) members). (3.22 release)
-   - **DEPENDENCY:** We can't do this until we can have [private code on Sourcegraph Cloud](../cloud/index.md#private-code-on-sourcegraph-cloud)
-1. We advertise a bounty for each unique vulnerability that allows an unauthorized person to gain access to our [test security repository](https://github.com/sourcegraph/security-test/blob/main/README.md) on Sourcegraph Cloud. (3.23 release)
-   - **DEPENDENCY:** This milestone depends on milestone 3 of [Visibility into Sourcegraph Cloud's attack surface](#visibility-into-sourcegraph-clouds-attack-surface). We should detect and fix the obvious issues ourselves before we start to invite others to attack us with large bounties.
-1. We run a time-bound capture the flag event where there are larger bounties for being able to gain access to our [test security repository](https://github.com/sourcegraph/security-test/blob/main/README.md) on Sourcegraph Cloud. (3.24 release)
-1. We host our private [infrastructure repository](http://github.com/sourcegraph/infrastructure) on Sourcegraph Cloud. (TBD, somewhat dependent on outputs of CtF)
-1. We allow paying customers to host their private code on Sourcegraph Cloud. (TBD, somewhat dependent on outputs of CtF)
+2. All compute nodes, repositories, and containers, as well as our cloud deployment, are continuously scanned for known security vulnerabilities and the security team is alerted as vulnerabilities are found. 
+   1. We have resolved compute vulnerabilities with a high severity, or a CVSS score of at least 4.0.
+3. TODO: Do we need additional logs for cloud GA? I would argue [GSuite log ingestion](https://github.com/sourcegraph/sourcegraph/issues/17235) and [org level GCP audit logs](https://github.com/sourcegraph/sourcegraph/issues/17335)
+4. We have confidence that our centralized logging is production-ready, and we have relevant tooling and testing environments configured.
+5. **Sourcegraph Cloud is Generally Available for user to add private code**
+6. We're confident that we're ingesting logs from all services that are tightly coupled to our Cloud deployment. This is anything that directly interacts with our cloud deployment with elevated privileges, or directly allows authentication to these services.
```

Comment:
somewhere after Cloud GA the following items are applicable:
- We continuously scan our source code for vulnerabilities (SAST) and have processes to triage and patch bugs.
- We continuously monitor our 3rd party dependencies for vulnerabilities and outdated versions.

Reply:
Good call. Added. Also pared down the pre-cloud GA work further to precisely mirror what we have in productboard.